### ISSUE TITLE
Wildcard search is not working in 2022 Release Wave 2

### ISSUE THREAD

**Issue by khoait** [MicrosoftDocs/powerapps-docs#3721]
Hi,

Wildcard search stops working after updating to 2022 Release Wave 2.

<img width="161" alt="image" src="https://user-images.githubusercontent.com/8686497/194537990-32ef0cc7-33aa-4140-b6d1-f6ada1d275c8.png">

This feature is helpful and is widely used by users. I'm not sure if the issue is a bug or a change but it's causing a lot of frustration after updating to Release Wave 2.

Is there a way to enable it again or this page should be updated to reflect the latest behavior with a warning of the change?

Kind regards,

Khoa Nguyen

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 425638fc-2f3d-6628-e5b1-46eca9ee2682
* Version Independent ID: 4c86534a-c032-6e63-a118-87ff8aaa42bf
* Content: [Quick Find - Power Apps](https://learn.microsoft.com/en-us/power-apps/user/quick-find)
* Content Source: [powerapps-docs/user/quick-find.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/user/quick-find.md)
* Service: **powerapps**
* Sub-service: **end-user**
* GitHub Login: @mduelae
* Microsoft Alias: **mkaur**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @khoait. This issue has been assigned to the doc author for review.

**Comment by khoait** [comment]
I found the problem. Looks like when enabling Dataverse Search, it doesn't respect the current value of "Prevent slow keyword filter for quick find terms on view pages" which is off.

I needed to turn it on and Save settings, then turn it off and Save settings for it to allow wildcard search.

![image](https://user-images.githubusercontent.com/8686497/194829425-a0600263-52fa-4547-99af-8debdc827e2f.png)

![image](https://user-images.githubusercontent.com/8686497/194829832-a065f059-382e-47bf-acaf-941c3cc4a0d1.png)
